ads/google/doubleclick.js

```diff
@@ -277,7 +280,14 @@ export function selectGptExperiment(data) {
  */
 export function writeAdScript(global, data, gptFilename) {
   const url =
-  `https://www.googletagservices.com/tag/js/${gptFilename || 'gpt.js'}`;
+        `https://www.googletagservices.com/tag/js/${gptFilename || 'gpt.js'}`;
+  const hasUSDRD = data.useSameDomainRenderingUntilDeprecated != undefined;
+  if (hasUSDRD) {
+    user().warn(TAG, 'useSameDomainRenderingUntilDeprecated will no longer ' +
```

Comment:
I think you only need do this in doubleclick config otherwise it will appear twice

Reply:
done